app/components/autocomplete/at_mention_group/at_mention_group.js

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {
+    Text,
+    View,
+} from 'react-native';
+import Icon from 'react-native-vector-icons/FontAwesome';
+
+import TouchableWithFeedback from 'app/components/touchable_with_feedback';
+import {makeStyleSheetFromTheme, changeOpacity} from 'app/utils/theme';
+
+export default class GroupMentionItem extends PureComponent {
+    static propTypes = {
+        completeHandle: PropTypes.string.isRequired,
+        onPress: PropTypes.func.isRequired,
+        theme: PropTypes.object.isRequired,
+    };
+
+    completeMention = () => {
+        const {onPress, completeHandle} = this.props;
+        onPress(completeHandle);
+    };
+
+    render() {
+        const {
+            completeHandle,
+            theme,
+        } = this.props;
+
+        const style = getStyleFromTheme(theme);
+
+        return (
+            <TouchableWithFeedback
+                onPress={this.completeMention}
+                style={style.row}
+                type={'opacity'}
+            >
+                <View style={style.rowPicture}>
+                    <Icon
```

Comment:
Was this icon approved by UX? Same for the styles?

Reply:
I think it's the same as special mentions since that's how it's implemented on WebApp side of things and that came from our UX designs.
![image](https://user-images.githubusercontent.com/17804942/78809371-d1312100-7994-11ea-8c11-6689de2ae45a.png)

But if any doubts, I can tag Michael and have him verify.

Reply:
I’ll add UX review after everything else has been addressed

Reply:
This is fine for now since it follows the pattern for @here, @channel, etc. 

Once we update the icons and update to the new mobile autocomplete we'll follow the pattern that's being established on the new webapp autocomplete (which can be seen on community-daily).
![image](https://user-images.githubusercontent.com/2040554/84950524-107a9d00-b0bd-11ea-8589-f4406c36e929.png)